docs/modules/ROOT/pages/how-tos/generic-authentication.adoc

```diff
@@ -0,0 +1,25 @@
+= Generic Restic Environment Variables
```

Comment:
Please rename the file so it matches the title somewhat, e.g. `generic-env-vars.adoc` (don't forget to update the nav link as well, see https://github.com/k8up-io/k8up/blob/master/docs/modules/ROOT/nav.adoc, I can't suggest edits in unchanged files)